packages/react-provider/src/index.tsx

```diff
@@ -63,6 +66,28 @@ export const CourierProvider: React.FunctionComponent<ICourierContext> = ({
     middleware,
   });
 
+  useEffect(() => {
+    if (_transport) {
+      // this means the transport was passed in and we shouldn't subscribe
+      return;
+    }
+
+    if (!transport || !userId) {
+      return;
+    }
+
+    const courierTransport = transport as CourierTransport;
```

Comment:
https://github.com/trycourier/courier-react/pull/63/files#r621480088

Reply:
whats this?

Reply:
this was a suggestion to avoid explicit casting transport to `CourierTransport`

Reply:
is there a differet way?